docs/deployment/scaling.md

```diff
@@ -60,5 +60,10 @@ target values for CPU and Memory utilization (in percent):
             cpu: 70
             memory: 90
 ```
-The number of [Processes](/reference/primitives/app/process) will be continually adjusted to maintain
-your target metrics.
\ No newline at end of file
+The number of [Processes](/reference/primitives/app/process) will be continually adjusted to maintain your target metrics.
+
+You must consider that the targets for CPU and Memory use the pods' limits to calculate the utilization percentage. So if you set the target for CPU as `70` and have two pods, it will trigger the HPA only if the utilization percentage sum divided by the pods' count is bigger than 70%. The desired replicas will be calculated to satisfy the percentage. Being the `currentMetricValue` computed by taking the average of the given metric across all Pods in the HorizontalPodAutoscaler's scale target.
+
+```html
+desiredReplicas = ceil[currentReplicas * ( currentMetricValue / desiredMetricValue )]
```

Comment:
Can we try rewriting this without using Kubernetes primitives?

>You must consider that the targets for CPU and Memory use the service replicas limit to calculate the utilization percentage. So if you set the target for CPU as `70` and have two replicas, it will trigger the Auto Scale only if the utilization percentage sum divided by the replica's count is bigger than 70%. The desired replicas will be calculated to satisfy the percentage. The current target value is computed by taking the average of the given metric across all service replicas